sync/src/utils.rs

```diff
@@ -0,0 +1,49 @@
+use crate::{Status, StatusCode};
+use ckb_metrics::metrics;
+use ckb_network::{CKBProtocolContext, PeerIndex, ProtocolId, SupportProtocols};
+use ckb_types::packed::{RelayMessage, SyncMessage};
+use ckb_types::prelude::*;
+
+pub(crate) fn send_message<Message: Entity>(
+    protocol_id: ProtocolId,
+    nc: &dyn CKBProtocolContext,
+    peer_index: PeerIndex,
+    message: &Message,
+) -> Status {
+    let name = message_name(protocol_id, message);
+    if let Err(err) = nc.send_message(protocol_id, peer_index, message.as_bytes()) {
+        return StatusCode::Network.with_context(format!("Send {}: {:?}", name, err));
```

Comment:
can we add a `debug!` log here? then we don't need check status every time after call this function

Reply:
Thanks. It should add a debug log here as you say. I will update it later.